core/cap-0038.md

```diff
@@ -786,30 +796,30 @@ if `line.type() == ASSET_TYPE_POOL_SHARE` and
 The behavior of `ChangeTrustOp` is changed for all trust line types
 
 If `line.type() != ASSET_TYPE_POOL_SHARE` then
-- If trust line is being deleted but `numPoolsInvolved != 0`,
+- If the asset trust line is being deleted but `liquidityPoolUseCount != 0`,
 return `CHANGE_TRUST_CANNOT_DELETE`.
 
 If `line.type() == ASSET_TYPE_POOL_SHARE` then
 - If any of the trust lines for an asset in the pool are missing,
 return `CHANGE_TRUST_TRUSTLINE_MISSING`.
-- The referenced trust line `tl` has
+- The referenced pool share trust line `tl` has
 `tl.asset.liquidityPoolID() == SHA256(line.liquidityPool())`
-- If a trust line is created, no flags are set.
+- If a pool share trust line is created, no flags are set.
```

Comment:
I think we should refactor this a bit. Can you group all the things that happen when creating into a single bullet (you should use sub-bullets)? Similarly, group all the things that happen on erasure. I think that'll make it a lot easier to understand and implement.

Reply:
Yeah my bad. Nico mentioned this but I missed it.

Reply:
Fixed